ci/test_wheel_deps_wheels.sh

```diff
@@ -25,16 +25,18 @@ test_dir = root + \"numba/cuda/tests/test_binary_generation/\"
 print(test_dir)
 "
 
-NUMBA_CUDA_TEST_BIN_DIR=$(python -c "$PY_SCRIPT")
```

Comment:
This moves the building of the tests but not the logging step associated with it.